compose/service.py

```diff
@@ -568,6 +574,7 @@ def connect_container_to_networks(self, container):
                     container.id,
                     network)
 
+            print('Connecting to {}'.format(network))
```

Comment:
Instead of `print`, we probably want to use `log.debug` here. It's not particularly useful information if the application runs normally.